app/assets/javascripts/users/settings/list_toggle.js.erb

```diff
@@ -0,0 +1,18 @@
+(function() {
+  'use strict';
+
```

Comment:
maybe you should also put $(document).ready here to be on the safe side

Reply:
:banana: